cabin-sorting/hackerSortingAlgo.ts

```diff
@@ -48,34 +54,45 @@ function matchAnswers () {
     // find backup cabin for hacker (in case the first choice fills up)
     const counterCopy = cabinScore.slice()
     counterCopy[maxIndex] = -1
-    hacker.secondAssignedCabin = cabinOptions[counterCopy.indexOf(Math.max(...counterCopy))]
+    hacker.secondAssignedCabin =
+      cabinOptions[counterCopy.indexOf(Math.max(...counterCopy))]
 
     // show their score for each cabin (for testing purposes)
     console.log(`${hacker.email}'s cabin counter: ${cabinScore}`)
   })
 }
-
+// test
 // Increment the given hacker's given cabinScore each time their answer
 // matches the Cabin's answer
 function hydrateCabinScore (hacker: any, cabinScore: number[]) {
-  answerList.forEach((cabin: any, cabinIndex: number) => {
-    for (let questionIndex = 0; questionIndex < QUESTIONS_SIZE; questionIndex++) {
+  let flag = false
+  for (let questionIndex = 0; questionIndex < QUESTIONS_SIZE; questionIndex++) {
+    flag = false
+    answerList.forEach((cabin: any, cabinIndex: number) => {
       if (
-        cabin['question' + questionIndex.toString()] ===
-        hacker['question' + questionIndex.toString()]
+        cabin['question' + questionIndex].toLowerCase().trim() ===
+        hacker['question' + questionIndex].toLowerCase().trim()
       ) {
         cabinScore[cabinIndex]++
+        flag = true
       }
+    })
+    if (!flag) {
+      console.log('THIS PERSON IS WRONG FOR SOME REASON?!?!?!')
+      console.log('\n' + hacker.email)
+      console.log(hacker['question' + questionIndex])
+      console.log(questionIndex)
     }
-  })
+  }
+  return 0
 }
 
 // Print to the console each hacker's information including their top
 // two cabin assignments
 function printMembers () {
   hackerList.forEach((member) => {
     console.log(
-      `ID: ${member.id}
+      `ID: ${member.id || member._id}
```

Comment:
both fields shouldn't exist on `member` so use just `_id` and delete `id` (ensure it does not break implementation)